docs/howtos/cert-manager.md

```diff
@@ -47,7 +50,7 @@ The DNS-01 challenge verifies domain ownership by proving you have control over
 
 1. Create the IAM policy specified in the cert-manager [AWS Route53](https://cert-manager.readthedocs.io/en/latest/tasks/acme/configuring-dns01/route53.html) documentation.
 
-2. Note the `accessKeyID` and create a secret named `prod-route53-credentials-secret` holding the `secret-access-key`.
+2. Note the `accessKeyID` and create a secret named `cert-manager-route-53` holding the `secret-access-key`.
```

Comment:
What is the reason for changing the name of the secret?

Reply:
It's just what I had in my cluster-issuer.yaml when I copied and pasted it, I'll change it back.